docs/src/user-guide/command-line-interface.md

```diff
@@ -186,35 +139,67 @@ npx eslint --global require --global exports:true
 
 #### `--parser`
 
-This option allows you to specify a parser to be used by ESLint. By default, `espree` is used.
+This option allows you to specify a parser to be used by ESLint.
+
+* **Argument Type**: String. Parser to be used by ESLint.
+* **Multiple Arguments**: No
+* **Default Value**: `espree`
+
+##### `--parser` example
+
+```shell
+# Use TypeScript ESLint parser
+npx eslint --parser @typescript-eslint/parser files.ts
+```
 
 #### `--parser-options`
 
-This option allows you to specify parser options to be used by ESLint. Note that the available parser options are determined by the parser being used.
+This option allows you to specify parser options to be used by ESLint. The available parser options are determined by the parser being used.
+
+* **Argument Type**: Key/Value pair. Separated by colon (`:`).
+* **Multiple Arguments**: Yes
 
-Examples:
+##### `--parser-options` example
 
 ```shell
-echo '3 ** 4' | npx eslint --stdin --parser-options=ecmaVersion:6 # fails with a parsing error
-echo '3 ** 4' | npx eslint --stdin --parser-options=ecmaVersion:7 # succeeds, yay!
+echo '3 ** 4' | npx eslint --stdin --parser-options ecmaVersion:6 # fails with a parsing error
+echo '3 ** 4' | npx eslint --stdin --parser-options ecmaVersion:7 # succeeds, yay!
 ```
 
 #### `--resolve-plugins-relative-to`
 
-Changes the folder where plugins are resolved from. By default, plugins are resolved from the current working directory. This option should be used when plugins were installed by someone other than the end user. It should be set to the project directory of the project that has a dependency on the necessary plugins. For example:
+Changes the folder where plugins are resolved from.
+
+* **Argument Type**: File path.
+* **Multiple Arguments**: No
+* **Default Value**: By default, plugins are resolved from the current working directory.
+
+This option should be used when plugins were installed by someone other than the end user. It should be set to the project directory of the project that has a dependency on the necessary plugins. If you use npm to install plugins, the path ends with `/node_modules/`.
+
+For example:
 
 * When using a config file that is located outside of the current project (with the `--config` flag), if the config uses plugins which are installed locally to itself, `--resolve-plugins-relative-to` should be set to the directory containing the config file.
 * If an integration has dependencies on ESLint and a set of plugins, and the tool invokes ESLint on behalf of the user with a preset configuration, the tool should set `--resolve-plugins-relative-to` to the top-level directory of the tool.
 
-### Specifying rules and plugins
+##### `--resolve-plugins-relative-to` example
+
+```sh
+npx eslint --config ~/personal-eslintrc.js \
+--resolve-plugins-relative-to /usr/local/lib/node_modules/
+```
+
+### Specify rules and plugins
```

Comment:
If we want to rename group headings, I think we should do the same in [`lib/options.js`](https://github.com/eslint/eslint/blob/f5808cb51529174a67b4938223f06435ad6d5118/lib/options.js#L180) so that the output of `eslint -h` is consistent with the CLI reference documentation.

Reply:
didn't realize there was congruency btwn this page and the headers there. i'll update those strings as part of this PR.

Reply:
i tried to add the updates to the strings in `lib/options.js`, but the pre-commit hook is failing with the following error. any ideas on how to fix?

```
 > running pre-commit hook: lint-staged
✔ Preparing...
⚠ Running tasks...
  ❯ Running tasks for *.js
    ✖ eslint --fix [FAILED]
  ↓ No staged files match *.md [SKIPPED]
  ↓ No staged files match docs/src/rules/*.md [SKIPPED]
  ↓ No staged files match docs/**/*.svg [SKIPPED]
↓ Skipped because of errors from tasks. [SKIPPED]
✔ Reverting to original state because of errors...
✔ Cleaning up...

✖ eslint --fix:

Oops! Something went wrong! :(

ESLint: 8.28.0

Error: Cannot find module 'is-path-inside'
Require stack:
- /Users/benperlmutter1/Library/Mobile Documents/com~apple~CloudDocs/eslint/lib/eslint/eslint-helpers.js
- /Users/benperlmutter1/Library/Mobile Documents/com~apple~CloudDocs/eslint/lib/eslint/flat-eslint.js
- /Users/benperlmutter1/Library/Mobile Documents/com~apple~CloudDocs/eslint/lib/eslint/index.js
- /Users/benperlmutter1/Library/Mobile Documents/com~apple~CloudDocs/eslint/lib/cli.js
- /Users/benperlmutter1/Library/Mobile Documents/com~apple~CloudDocs/eslint/bin/eslint.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/benperlmutter1/Library/Mobile Documents/com~apple~CloudDocs/eslint/lib/eslint/eslint-helpers.js:21:22)
    at Module._compile (node:internal/modules/cjs/loader:1097:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1149:10)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:999:19)

pre-commit hook failed (add --no-verify to bypass)
```

Reply:
You probably have outdated `node_modules`, so they don't contain all the currently needed dependencies. Try removing the `node_modules` directory and then run `npm install` to generate it again.

Reply:
yep ,that did it. updated the help output text in [19117e9](https://github.com/eslint/eslint/pull/16563/commits/19117e964a93068d0c86a85197b81a53988b6fc6)